CMakeLists.txt

```diff
@@ -527,7 +527,7 @@ PRIVATE
 #        "CMAKE_CXX_COMPILER_ID = ${CMAKE_CXX_COMPILER_ID} "
 #        "CMAKE_CXX_COMPILER_VERSION = ${CMAKE_CXX_COMPILER_VERSION}"
 #)
-if(${CMAKE_CXX_COMPILER_ID} MATCHES "MSVC")
+if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
```

Comment:
Ahh! Appveyor likely has an older version of CMake than GitHub Actions in its VS 201[579] images. Certainly true for the VS201[57] images.